File: Frontend/trackr-administrador/src/app/shared/http/facturacion/sat-forma-pago.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Observable } from 'rxjs';
import { SatFormaPago } from '@models/facturacion/sat-forma-pago';

@Injectable()
export class SatFormaPagoService {
  private dataUrl = 'satFormaPago/';

  constructor(public http: HttpClient) { }

  public consultarParaSelector() {
    return this.http.get<SatFormaPago[]>(this.dataUrl + 'consultarParaSelector');
  }

}

File: Backend/TrackrAPI/TrackrAPI/Services/Facturacion/SatFormaPagoService.cs
using TrackrAPI.Models;
using TrackrAPI.Repositorys.Facturacion;

namespace TrackrAPI.Services.Facturacion
{
    public class SatFormaPagoService
    {
        private ISatFormaPagoRepository satFormaPagoRepository;

        public SatFormaPagoService(ISatFormaPagoRepository satFormaPagoRepository)
        {
            this.satFormaPagoRepository = satFormaPagoRepository;
        }

        public IEnumerable<SatFormaPago> ConsultarParaSelector()
        {
            return satFormaPagoRepository.ConsultarParaSelector();
        }

    }
}
